x-pack/plugins/timelines/public/mock/t_grid.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TGridIntegratedProps } from '../components/t_grid/integrated';
+import { mockBrowserFields, mockDocValueFields } from './browser_fields';
+import { mockDataProviders } from './mock_data_providers';
+import { ColumnHeaderOptions, TimelineId } from '../../common';
+import { mockIndexNames, mockIndexPattern } from './index_pattern';
+
+const columnHeaders: ColumnHeaderOptions[] = [
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: 'Status',
+    id: 'kibana.rac.alert.status',
```

Comment:
These should be `kibana.alert.{insert-field-here}`. We're using `workflow_status` now for the status of the alert.

You can just use the constants and import them from `@kbn/rule-data-utils`